### PR TITLE
[Mistweaver] Add Talent Summary Panel

### PR DESCRIPTION
**Pull request by vohrr**
- Added a talent summary statistic to the core section of analysis - conditionally loading and sorting talents by their contribution
- Updated & reformatted the Average Renewing Mist per Vivify statistic for consistency
- Updated Misty Peaks breakdown to include contributions from  mists of life renewing mists

Talent Summary:
image

Old rems per vivify stat: 
image

New rems per vivify:
image

### CODE REVIEW

File: src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx [hunk 1]
```diff
@@ -162,32 +162,33 @@ class Vivify extends Analyzer {
...
+        tooltip={
+          <>
+            Healing Breakdown:
+            <ul>
+              <li>
+                {formatNumber(this.mainTargetHealing + this.cleaveHealing)} overall healing from
+                Vivify.
+              </li>
+              <li>
+                {formatNumber(this.cleaveHealing)} portion of your Vivify healing to REM targets.
```
Comment (trevorm4) [inline src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx]:
```suggestion
                {formatNumber(this.cleaveHealing)} portion of your <SpellLink id={SPELLS.VIVIFY.id}/> healing to <SpellLink id={TALENTS_MONK.RENEWING_MIST_TALENT.id}/> targets.
```

File: src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx [hunk 2]
```diff
@@ -162,32 +162,33 @@ class Vivify extends Analyzer {
...
-            Avg REMs per Cast
-          </TooltipElement>
+        size="flexible"
+        tooltip={
+          <>
+            Healing Breakdown:
+            <ul>
+              <li>
+                {formatNumber(this.mainTargetHealing + this.cleaveHealing)} overall healing from
+                Vivify.
```
Comment (trevorm4) [inline src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx]:
```suggestion
                <SpellLink id={SPELLS.VIVIFY.id}/>.
```

**Review by trevorm4 — APPROVED** [review]
Small nits, approving

File: src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx (hunk 1 above)
Comment (vohrr) [reply src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx]: Updated

**Pull request merged**